client/src/GlobalStyles.js

```diff
@@ -9,6 +9,8 @@ body {
   display: flex;
   justify-content: center;
   font-family: sans-serif;
+  color: #fff;
+  background-color: #1e1e1e;
```

Comment:
you might think about setting color variables

Reply:
I have planned that for later when I get into the details of the design.